frontend/src/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,19 @@
+import { NavLink } from 'react-router-dom';
+
+import { AppRoute } from '~/libs/enums/app-route.enum';
+
+const NotFound = (): JSX.Element => (
+  <div>
+    <h2>
+      Oops, the dreaded <span>404</span>.
+    </h2>
+
+    <p>Let&apos;s get you back on track.</p>
+
+    <p>
+      <NavLink to={AppRoute.ROOT}>Back to Home page</NavLink>
```

Comment:
create custom component wrapper Link which will get children, to, and classname through props. This Link component will be used throughout the project

Reply:
add reusable component Link instead just NavLink. added props className to this component